dotCMS/src/integration-test/java/com/dotcms/util/TestDataSourceProvider.java

```diff
@@ -11,50 +12,12 @@
 
 public class TestDataSourceProvider implements DotDataSourceStrategy {
 
-    private Properties prop;
-
     @Override
     public DataSource apply() {
 
-        String dbType = "postgres.";
-
-        if (System.getProperty("databaseType")!=null){
-            dbType = System.getProperty("databaseType") + ".";
-        } else if(System.getenv("databaseType")!=null){
-            dbType = System.getenv("databaseType") + ".";
-        }
-
-        loadProperties();
-
-        // https://github.com/brettwooldridge/HikariCP#configuration-knobs-baby
-        HikariConfig config = new HikariConfig();
+        HikariConfig config = new HikariConfig(getProperties());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'config' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5566474)